src/middlewared/middlewared/alert/service/any_webhook.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from middlewared.alert.base import ThreadedAlertService
+from middlewared.schema import Dict, Str
+from middlewared.utils.network import INTERNET_TIMEOUT
+
+class AnyWebhookAlerteService(ThreadedAlertService):
```

Comment:
It would be nice to use `aiohttp` client and avoid using threads

Reply:
Maybe but I have to admit that I'm really not fluent in Python and I used the other alert services as example.